biolink-model.yaml

```diff
@@ -202,7 +202,105 @@ enums:
         description: nucleic acid
       "AA":
         description: amino acid
-        
+
+  drug_availability_enum:
+    description: >-
+    permissible_values:
+      "over the counter":
+        description: >-
+          chemical entity is available over the counter without a prescription.
+      "prescription":
+        description:
+          chemical entity is available by prescription.
+
+  FDA_approval_status_enum:
+    description: >-
+    permissible_values:
+      "FDA Approval Phase 1":
```

Comment:
"FDA Approval Phase 1" is easily confused with "FDA Clinical Research Phase 1", I suggest to call this phase "Discovery & Development Phase"

Reply:
done